cmd/commands/commands_test.go

```diff
@@ -120,3 +120,57 @@ func TestParseTimeLockDelta(t *testing.T) {
 		}
 	}
 }
+
+// TestReplaceCustomData tests that hex encoded custom data can be formatted as
+// JSON in the console output.
+func TestReplaceCustomData(t *testing.T) {
```

Comment:
nit: add a new test `TestReplaceCustomDataErr` to check the error case?

Reply:
Good idea! Added some negative test cases instead.